Nodejs/Product/Nodejs/Debugger/NodeDebugger.cs

```diff
@@ -530,15 +530,13 @@ public NodeDebugger(Uri debuggerEndpointUri, int id)
             foreach (NodeModule module in modules) {
                 NodeModule newModule;
                 if (GetOrAddModule(module, out newModule)) {
-                    if (newModule.FileName != newModule.JavaScriptFileName) {
```

Comment:
Here's my rough understanding of the problem.

First is the case issue. Without the correct case, we fail to set breakpoints properly at all. Fix this, and we run into another issue: Node 6 does not seem to handle the way we set breakpoints during initial program startup. This manifests itself as a race condition, which explains why see the bug more often in the more complex express apps and more often in external files.

The root cause of the second issue could be somewhere in our code still, but the casing fix alone is was not enough to consistently fix the bug for me. This change works by making us reset the breakpoints when we get an "afterCompile" event for a file, brute forcing the breakpoint into existence.